.env.sample

```diff
@@ -11,4 +11,5 @@ FIREBASE_DYNAMIC_LINK_URL=
 FIREBASE_API_KEY=
 BETTER_WEB_APP_URL=
 BETTER_APP_STORE_ID=
-DATABASE_URL=
\ No newline at end of file
+DATABASE_URL=
+SEGMENT_WRITE_KEY=
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [15-16]

The addition of `SEGMENT_WRITE_KEY` is a good move for integrating Segment analytics. However, it's important to ensure that this key is kept secure and not exposed in any public repositories or logs. If the application is open source, consider adding instructions in the README file about how to obtain and set up this key.
```diff
  DATABASE_URL=
+ SEGMENT_WRITE_KEY= # Add your Segment Write Key here
\ No newline at end of file
```
Also, please make sure to add this new environment variable to all environments (development, staging, production) where the application is deployed.


<!-- This is an auto-generated comment by OSS CodeRabbit -->